website/docs/troubleshooting/troubleshooting-podman.md

```diff
@@ -128,6 +128,40 @@ Podman Desktop might display "No Containers" as shown below, even if there are a
    $ podman machine init
    ```
 
+## Podman Desktop is failing to display the images or containers from a rootlful Podman machine
```

Comment:
```suggestion
## Podman Desktop is failing to display the images or containers from a rootful Podman machine
```

looks like there is a typo (don't know how it's not reported as a warning in checks)